src/main/java/animals/Application.java

```diff
@@ -0,0 +1,47 @@
+package animals;
+
+import java.io.IOException;
+
+public final class Application extends TextInterface implements Runnable {
```

Comment:
Can you use the same pattern for all of your classes? I mean add an empty line after class declaration.

Reply:
Done